xarray/core/combine.py

```diff
@@ -390,6 +410,15 @@ def combine_nested(
         - 'override': if indexes are of same size, rewrite indexes to be
           those of the first object with that dimension. Indexes for the same
           dimension must have the same size in all objects.
+    combine_attrs : {'drop', 'identical', 'no_conflicts', 'override'}, optional
```

Comment:
This could note the default too.

```suggestion
    combine_attrs : {'drop', 'identical', 'no_conflicts', 'override'},
                    default 'drop'
```